### PR TITLE
Fix for doc issues

### PR DESCRIPTION
**Pull request by joel-hamill**
I believe this is generated content, but here is a start on what will fix the doc issues.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (joel-hamill) [inline docs/index.rst]: a space required at the top of the file, or and RST anchor (e.g. `.. _confluent-kafka-index:`

**Comment by joel-hamill** [comment]
@ewencp is this index.rst page auto-generated - or is it hand-coded?

**Comment by edenhill** [comment]
@rnpridgeon Can you verify that the removal of `:members:` does not actually remove the class member listings from the generated docs?

**Comment by ewencp** [comment]
@joel-hamill afaik it is manually generated. the autogenerated stuff is just via the `automodule` directive.

**Comment by joel-hamill** [comment]
@ewencp well then this is ridiculously easy to fix. 🤷‍♂

**Pull request merged**